package.json

```diff
@@ -15,13 +15,13 @@
     "update-actions-build": "node actions/build.js"
   },
   "dependencies": {
-    "@actions/core": "^1.2.7",
+    "@actions/core": "^1.10.0",
     "@actions/github": "^2.2.0",
-    "axios": "^0.21.1",
-    "dotenv": "^9.0.2",
-    "google-auth-library": "^7.0.4",
-    "googleapis": "^73.0.0",
```

Comment:
This version change seems to be a big jump. Were u able to check the change logs if any functions we use have been changed?

Reply:
@gp201 There are some of the APIs that Oppiabot seems to use that are listed under breaking changes between the two versions at https://github.com/googleapis/google-api-nodejs-client/blob/main/CHANGELOG.md but I'm not sure if they would affect the functionality of the codebase